cmd/storage-datatypes.go

```diff
@@ -153,6 +153,9 @@ type FileInfo struct {
 	VersionPurgeStatus            VersionPurgeStatusType
 
 	Data []byte // optionally carries object data
+
+	NumVersions      int
```

Comment:
As per the note, this breaks API and requires a version bump.

Reply:
> As per the struct note, this breaks API and requires a version bump.

Done, I bumped storageRESTVersion in cmd/storage-rest-common.go